### PR TITLE
Update Set-SPOTenantRenameSitePrioritization.md - link fix from publi…

### PR DESCRIPTION
**Pull request by American-Dipper**
…sh PR

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Allows for the specified site to be prioritized.
 
 As part of [Advanced Tenant Rename](/sharepoint/change-your-sharepoint-domain-name#advanced-tenant-rename-preview), organizations can prioritize up to 4,000 sites for initial execution among all other sites in your organization as part of the overall rename operation.
 
-It is possible to start prioritizing sites only once the rename operation has been scheduled using the [Start-SPOTenantRename](/start-spotenantrename?view=sharepoint-ps) cmdlet.
+It is possible to start prioritizing sites only once the rename operation has been scheduled using the [Start-SPOTenantRename](Start-SPOTenantRename.md?view=sharepoint-ps) cmdlet.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md:27]: `start-spotenantrename` -- without the `.md` extension.

Use the [preview URL](https://review.learn.microsoft.com/en-us/powershell/module/sharepoint-online/set-spotenantrenamesiteprioritization?view=sharepoint-ps&branch=pr-en-us-595) to verify: current link (with the `.md`) will give you a 404

**Review by samkabue — CHANGES_REQUESTED** [review]

**Comment by samkabue** [comment]
@shdyas I see this PR was just merged, but it still has issues -- as can be seen from the last Preview URL, that link is still broken.

**Comment by shdyas** [comment]
@samkabue when I click on the same link it resolves? 

https://review.learn.microsoft.com/en-us/powershell/module/sharepoint-online/set-spotenantrenamesiteprioritization?branch=main&branchFallbackFrom=pr-en-us-595&view=sharepoint-ps

**Comment by samkabue** [comment]
@shdyas great! sounds good then. The PR preview link must have been bad then.

**Pull request merged**